src/Sidekick/Windows/LeagueOverlay/LeagueOverlayView.xaml.cs

```diff
@@ -70,9 +77,93 @@ public LeagueOverlayView()
             };
 
             tabControlLeagueOverlay.SelectionChanged += TabControlLeagueOverlay_SelectionChanged;
+            tabControlLeagueOverlay.MouseWheel += TabControlLeagueOverlay_MouseWheel;
             CurrentPage = tabItemIncursion;
         }
 
+        public bool IsDisplayed => Visibility == Visibility.Visible;
+
+        private void TabControlLeagueOverlay_MouseWheel(object sender, System.Windows.Input.MouseWheelEventArgs e)
```

Comment:
You have a bug here? It will get out of bounds the way it is now.
```c#
if (e.Delta > 0 && tabControlLeagueOverlay.SelectedIndex < tabControlLeagueOverlay.Items.Count)
{
    tabControlLeagueOverlay.SelectedIndex++;
}
else if (e.Delta < 0 && tabControlLeagueOverlay.SelectedIndex > 0)
{
    tabControlLeagueOverlay.SelectedIndex--;
}
```

Reply:
Fixed Scrolling Bug